src/features/formData/FormDataWrite.tsx

```diff
@@ -235,14 +235,15 @@ function useFormDataSaveMutation() {
           if (!url) {
             throw new Error(`Cannot patch data, url for dataType '${dataType}' could not be determined`);
           }
-          const { newDataModel, validationIssues } = await doPatchFormData(url, {
+          const { newDataModel, validationIssues, instance } = await doPatchFormData(url, {
             patch,
             // Ignore validations that require layout parsing in the backend which will slow down requests significantly
             ignoredValidators: IgnoredValidators,
           });
           return {
             newDataModels: [{ dataType, data: newDataModel, dataElementId }],
             validationIssues,
+            instance,
```

Comment:
Looks good, men nå har ikke jeg helt kontekst her. Bruker vi faktisk `instance` noe sted? 😊

Reply:
Yes, vi oppdaterer instansdata dersom backend returnerer det, det ble lagt til for nye multi-patch først, men dette bare legger det til for vanlig patch også. Det er relatert til dette https://github.com/Altinn/app-frontend-react/issues/1335

Reply:
I see 👌